proposals/XXXX-efficient-token-program.md

```diff
@@ -37,6 +37,29 @@ N/A.
 
 `p-token` is a like-for-like efficient re-implementation of the Token program. It is `no_std` (no heap memory allocations are made in the program) and uses zero-copy access for instruction and account data. Since it follows the same instructions and accounts layout, it does not require any changes to client code – it works as a drop-in replacement.
 
+Apart from the original SPL Token instructions, this proposal adds two additional instructions to the program:
+
+1. `withdraw_excess_lamports` (instruction discriminator `38`): allow recovering "bricked" SOL from mint accounts (e.g., USDC mint as `~323` SOL in excess). The logic of this instruction is the same as the current SPL Token-2022 instruction. 
```

Comment:
Important note: it also allows people to withdraw extra SOL from SPL-Token multisig accounts